src/core_plugins/kibana/public/management/sections/objects/components/objects_table/components/header/header.js

```diff
@@ -53,7 +59,16 @@ export const Header = ({
               data-test-subj="exportAllObjects"
               onClick={onExportAll}
             >
-              Export {filteredCount} {filteredCount === 1 ? 'object' : 'objects'}
+              <FormattedMessage
+                id="kbn.management.objects.objectsTable.header.exportButtonLabel"
+                defaultMessage="Export {filteredCount} {filteredCountDescription}"
```

Comment:
`object` and `objects` should be translated too, so please use pluralization here:
`Export {filteredCount, plural, one {# object} other {# objects} }`